src/main/java/jenkins/plugins/zulip/ZulipUtil.java

```diff
@@ -73,4 +76,69 @@ public static String expandVariables(@Nonnull Run<?, ?> run, @Nonnull TaskListen
         return expandedMessage;
     }
 
+    /**
+     * Helper method to display an item, optionally with links.
+     *
+     * @param item         The item to display.
+     * @param globalConfig Zulip global configuration.
+     * @param fullPath     Whether to display the full path including the display name of parent items
+     *                     ({@code true}) or just this item's display name ({@code false}).
+     * @param displayLinks Whether to display links to the item and (if relevant) its parent items.
+     * @return A string representing the item.
+     */
+    public static String displayItem(Item item, DescriptorImpl globalConfig, boolean fullPath, boolean displayLinks) {
+        StringBuilder builder = new StringBuilder();
+        // Don't call getUrl() unless necessary: the logic behind that method is complex.
+        displayObject(builder, item, displayLinks ? item.getUrl() : null, globalConfig, fullPath, displayLinks);
+        return builder.toString();
+    }
+
+    /**
+     * Helper method to display a model object, optionally with links.
+     *
+     * @param builder      The builder to append to.
+     * @param object       The object to display.
+     * @param globalConfig Zulip global configuration.
+     * @param fullPath     Whether to display the full path including the display name of parent items
+     *                     ({@code true}) or just this object's display name ({@code false}).
+     * @param displayLinks Whether to display links to the object and (if relevant) its parent items.
+     */
+    private static void displayObject(StringBuilder builder, ModelObject object, String url, DescriptorImpl globalConfig,
+                                      boolean fullPath, boolean displayLinks) {
+        // The only common interface between Item and ItemGroup is ModelObject, which doesn't define getParent,
+        // so we need to resort to instanceof + cast to crawl up the item tree.
+        if (fullPath && object instanceof Item) {
```

Comment:
Traversing all the way to the top will also print out the Jenkins instance itself which is something we most likely want to avoid.
![image](https://user-images.githubusercontent.com/17402486/135939601-41ccaf96-8614-45ff-a64a-88660802ef86.png)

One possibility is to add something like this to the beginning of the method:

```
if (object instanceof jenkins.model.Jenkins) {
    return;
}
```